stdlib/public/TensorFlow/Ops.swift

```diff
@@ -577,6 +577,38 @@ public extension Tensor where Scalar : FloatingPoint & Equatable {
   }
 }
 
+public extension Tensor where Scalar : FloatingPoint {
+  // TODO: std() should handle non floating point Tensors.
```

Comment:
Although I'm very tempted to just add the implementation for `Int` backed by `float64`, I'm not quite sure thats always the right answer. (Concretely, float64 is a performance anti-pattern for accelerators.) An alternative would be to make it generic, and require the users to specify the amount of precision they'd like, which could work (at the cost of extra cognitive overhead for users, which we could make go-away in the default case if we can implement defaults for generic type parameters). What do you think about that?

Reply:
I think making things generic is a great idea.

How about this?
```swift
extension Tensor where Scalar : Numeric {
  func standardDeviation<FloatingPointScalar>(
    alongAxes axes: Int32..., withScalarType: FloatingPointScalar.Type
  ) where FloatingPointScalar : TensorFlowFloatingPoint { ... }

  // Enabling `standardDeviation(withType: ...)` without specifying axes is important.
  // Perhaps this can be done with a default `[]` axes argument.
  // There may be a better way.
  func standardDeviation<FloatingPointScalar>(
    alongAxes axes: [Int32] = [], withScalarType: FloatingPointScalar.Type
  ) where FloatingPointScalar : TensorFlowFloatingPoint { ... }
}

// Usage:
let x: Tensor<Int32> = ...
x.standardDeviation(withScalarType: Float.self)
x.standardDeviation(alongAxes: 0, withScalarType: Float.self)
```

cc @rxwei

Reply:
Your examples above don’t have a return type. Did you mean it’s Tensor<FloatingPointScalar>?

Reply:
> Your examples above don’t have a return type. Did you mean it’s a Tensor of FloatingPointScalar?

Fixed, thanks!